src/dosemu-setupfreedos

```diff
@@ -111,7 +111,7 @@ if not Path(cmddir, 'fdconfig.sys').is_file():
     cmddir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'c')
 
 if not args.source:
-    shortname = 'freedos12'
+    shortname = 'freedos13'
```

Comment:
The whole PR seems fine. I modified a version of my proposed test PR to look at your PR branch, with the new 1.3 options, and all the 1.1, 1.2 and 1.3 choices passed